testdata/nested.json

```diff
@@ -0,0 +1,20 @@
+{
+  "string": {
+    "key": "a string",
+    "bool": true
+  },
+  "float": {
+    "nested": {
+      "key": 1.23
+    }
+  },
+  "strings": {
+    "nested": {
+      "key": [
+        "one",
+        "two",
+        "three"
+      ]
+    }
+  }
+}
```

Comment:
I opted for `x` rather than `strs` because that's what the testing flag is called.  
I agree `strs` is maybe more obvious, though, so let me know if you want me to change it to that explicitly.